edx_reverification_block/xblock/static/sass/main.scss

```diff
@@ -0,0 +1,150 @@
+$red: #D0021B;
+$light-gray: #DDDDDD;
+$light-blue: #009FE6;
+$blue: #126F9A;
+
+@mixin msg-box($highlight-color) {
+    padding: 20px 30px;
+    border-left: 5px solid $highlight-color;
+    border-top: 1px solid $highlight-color;
+    border-bottom: 1px solid $highlight-color;
+    border-right: 1px solid $highlight-color;
+    box-shadow: -1px 2px 3px $light-gray;
+}
+
+.hidden {
+    display: none
+}
+
+.reverify-clear-line {
+    clear: both;
+}
+
+.reverify-xblock {
+    .action {
+        font: 16px/1.3 "Open Sans","Helvetica Neue",Helvetica,Arial,sans-serif;
+        box-sizing: border-box;
+        cursor: pointer;
+        transition: color .25s ease-in-out, background .25s ease-in-out;
+        display: inline-block;
+        border: 1px solid #007aba;
```

Comment:
nitpick: the file starts with assigning some colours as variables, it would be cleaner to include all colours as variables.